src/datanode/src/server.rs

```diff
@@ -17,11 +17,12 @@ use std::net::SocketAddr;
 use std::sync::Arc;
 
 use common_runtime::Builder as RuntimeBuilder;
+use common_telemetry::tracing::log::info;
+use frontend::frontend::Mode;
```

Comment:
Move `Mode` to the server crate, it's not only used in the frontend. Not related to this PR, but recommend refactoring it.